doc/source/whatsnew/v1.0.0.rst

```diff
@@ -341,6 +341,7 @@ MultiIndex
 
 - Constructior for :class:`MultiIndex` verifies that the given ``sortorder`` is compatible with the actual ``lexsort_depth``  if ``verify_integrity`` parameter is ``True`` (the default) (:issue:`28735`)
 -
+- Using `.loc` with a `PeriodIndex` as a level in a `MultiIndex`, `.loc` could not raise exception when key is not matched (:issue:`20684`)
```

Comment:
you didn't make any code changes, is this a test only PR?

Reply:
@jreback 
Yes. Only add tests.
I try to change `PeriodIndex.get_loc` in two ways.
First, I try to remove `PeriodEngine.get_loc` in `PeriodIndex.get_loc`, so just compute `.get_loc` once, not several in try-except statement. I change several ways  But always stuck in test cases. Changing some test cases is inevitable. So I give it up this way.
Second, I try to change `PeriodEngine.get_loc` to accept `Period` object not only `Period.ordinal`. (Original `PeriodEngine.get_loc` only accept `Period.ordinal`) This way is success in that work successfully and pass all test cases. However Problem is most cases user don't use `Period` object using `PeriodIndex.get_loc` And `PeriodEngine.get_loc` can't accept "intolerance" argument. So Integrating `PeriodIndex.get_loc` can't be possible.
In conclusion, Refactoring `PeriodIndex.get_loc` fails. Someone who is good at programming more than me can do it maybe.

Reply:
@jreback 
And while i get review, someone change what i try to change

Reply:
ok, then don't need the whatsnew note. where was the change, can you point to a PR?